src/js/hooks/inboundV2/addItems/useInboundAddItemsForm.jsx

```diff
@@ -14,16 +14,16 @@ import stockMovementApi from 'api/services/StockMovementApi';
 import {
   STOCK_MOVEMENT_BY_ID,
   STOCK_MOVEMENT_ITEM_REMOVE,
-  STOCK_MOVEMENT_ITEMS,
   STOCK_MOVEMENT_REMOVE_ALL_ITEMS,
   STOCK_MOVEMENT_UPDATE_INVENTORY_ITEMS,
   STOCK_MOVEMENT_UPDATE_ITEMS,
   STOCK_MOVEMENT_UPDATE_STATUS,
 } from 'api/urls';
 import { STOCK_MOVEMENT_URL } from 'consts/applicationUrls';
-import InboundV2Step from 'consts/InboundV2Step';
+import InboundV2Step from 'consts/InboundStep';
 import RequisitionStatus from 'consts/requisitionStatus';
 import { DateFormat, DateFormatDateFns } from 'consts/timeFormat';
+import { OutboundWorkflowState } from 'consts/WorkflowState';
```

Comment:
I know they are almost the same (with outbound just having more states), but for clarity I'd create a similar one for the inbound.